addons/mail/models/ir_http.py

```diff
@@ -1,14 +1,23 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 import odoo
-from odoo import api, models, fields
+from odoo import models
 from odoo.http import request
+from odoo.addons.mail.controllers.webclient import WebclientController
 from odoo.addons.mail.tools.discuss import Store
 
 
 class IrHttp(models.AbstractModel):
     _inherit = 'ir.http'
 
+    def _initial_store_params(self):
+        return ["failures", "systray_get_activities", "init_messaging"]
```

Comment:
This is not the right approach. The params must code from the client not the server.
Depending on context (live chat in particular) we'll only want to fetch `init_messaging` and not the others.